src/plugins/kibana/public/dashboard/services/_saved_dashboard.js

```diff
@@ -56,6 +57,7 @@ module.factory('SavedDashboard', function (courier, config) {
     timeRestore: 'boolean',
     timeTo: 'string',
     timeFrom: 'string',
+    refreshInterval: 'string'
```

Comment:
This causes a `mapper_parsing_exception` on new Kibana installs because refreshInterval isn't actually a string. The problem is masked on existing Kibana installs because the mappings here only get created the first time a dashboard is saved. So on existing Kibana installs, the first time a refreshInterval is included with a dashboard object Elasticsearch just dynamically maps it as an object type field. It also persists fields that we probably don't really need to persist, like display name and angular's internal $$hashkey

What info do we really need to persist? Just the value in millis probably? If so we should probably map this field as an integer, and then either implement a custom serializer/deserializer (example [here](https://github.com/elastic/kibana/blob/master/src/ui/public/index_patterns/_index_pattern.js#L39)) or just pass the value property in dashboard/index.js instead of the entire object.